### DISCUSSION TITLE
Add a view in database to view rows of a selected table

### DISCUSSION THREAD

**Discussion by Brijendra-Singh2003** [supabase/supabase#20278] [Feature Requests]
I am not able to see what data is stored in my table. It only shows no of rows but has no button to view those rows.
at https://supabase.com/dashboard/project/[id]/database/tables

https://github.com/supabase/supabase/assets/113870500/b88b9ed8-55ba-49d2-a314-6b39d43a42c2

**Comment by Hallidayo** [comment]
Hi @Brijendra-Singh2003 -  I'll take a look at this one and add an option to view the table.
